R/02-importData-queryData.R

```diff
@@ -477,7 +477,8 @@ gptServer <- function(id, autoCompleteList, isActiveTab) {
                        prompt_content = paste("Write an SQL query to", input$gptPrompt),
                        temperature = input$temperature,
                        max_tokens = input$maxTokens,
-                       n = input$n
+                       n = input$n,
```

Comment:
I think this should be fixed to the default value 1 and should not be changed by the user

>  n: numeric (default: 1) specifying the number of completions per
          request (from the official API documentation: _How many chat
          completion choices to generate for each input message.
          **Note: Because this parameter generates many completions, it
          can quickly consume your token quota.** Use carefully and
          ensure that you have reasonable settings for max_tokens and
          stop._)